### DISCUSSION TITLE
Welcome to EfficientWord Discussions!

### DISCUSSION THREAD

**Discussion by aman-17** [Ant-Brain/EfficientWord-Net#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LANCE-PARTH** [comment]
Reallly dumb question but, when I train a wake word, will it only work well on my voice or will it pick up anyone (different accents, noise levels, etc) saying the word?

Additionally, does it run on low-power hardware such as ESP 32?

**Reply by aman-17** [reply]
It will pick up anyone (different accents, noise levels, etc) saying any word. I'm not sure how it will perform on ESP 32. Can you help us by trying it and letting us know?

**Comment by OnlinePage** [comment]
Hi been waiting for the updates , the improvised version when likely it is going to come??🤔

**Reply by TheSeriousProgrammer** [reply]
Currently just debugging some bugs, will release soon, may be in 2 to 3 days

**Comment by TheSeriousProgrammer** [comment]
Update , a beta version with new model is now released. IBM generator has been updated for the new demo api
The new model allows for a max input length of 1.5 secs to facilitate longer words, try it out

**Reply by OnlinePage** [reply]
![SpongebobHiiiiiGIF](https://user-images.githubusercontent.com/61045650/232060749-349948c6-eed0-48de-af1a-ddb5ea1efdcd.gif)

to try!!

**Reply by OnlinePage** [reply]
> Update , a beta version with new model is now released. IBM generator has been updated for the new demo api The new model allows for a max input length of 1.5 secs to facilitate longer words, try it out

Raise a issue related to models not found, do check out!
